File: assignment 10 - C# practice problems/19.cs
//Write a c# function to get the month name from a particular date.

using System;
namespace MyNameSpace
{
    class Program19
    {
        public static void GetMonthNameFromDateDriverFunction()
        {

            var monthStrings = new[]{"January", "February",
                         "March", "April", "May", "June", "July",
                         "August", "September", "October", "November",
                        "December"
            };

            Console.WriteLine("Enter dateInput in dd/mm/yyyy");
            var dateInput = Console.ReadLine();
            var split = dateInput.Split('/');

            Console.WriteLine("Month name is " + monthStrings[int.Parse(split[1]) - 1]);


        }
    }
}

File: assignment 3 - js practice problems/web-dev-starter/19.js
//Write a JavaScript function to get the month name from a particular dateInput.
let monthStrings = ["January", "February",
    "March", "April", "May", "June", "July",
    "August", "September", "October", "November",
    "December"
]

function getMonthNameFromDate(month) {
    return monthStrings[month - 1];
}

function getMonthNameFromDateDriverFunction() {
    let dateInput = prompt("Enter dateInput in dd/mm/yyyy");
    dateInput = dateInput.split("/");

    // let day = parseInt(dateInput[0]);
    let month = parseInt(dateInput[1]);
    // let year = parseInt(dateInput[2]);

    alert(getMonthNameFromDate(month));
}

getMonthNameFromDateDriverFunction();